.github/workflows/e2e-tests.yml

```diff
@@ -45,3 +45,10 @@ jobs:
       - name: Stop WordPress environment
         run: |
           npm run dev:stop
+
+      - name: Archive e2e results
+        uses: actions/upload-artifact@v3
+        if: failure()
+        with:
+          name: test-results
+          path: artifacts
```

Comment:
Can you please explain a bit where are we storing the results and how can we view it if needed?

Reply:
Yes! When E2E tests fail, and you go to the particular action (https://github.com/Parsely/wp-parsely/actions/runs/6099498942?pr=1850 for this PR), you get an Artifacts file that you can download and use for help in debugging.

Reply:
Added this in a "How has this been tested?" section in the PR's description.